src/styles/index.scss

```diff
@@ -1,3 +1,126 @@
+$mainAccent:#060b35;
+$secondary: #616070;
+$blueAccent: #00acdc;
+
+* {
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+.card {
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+
+  &__image {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin: 0 auto;
+  }
+
+  &__name {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    color: $mainAccent;
+    margin: 40px 0 0;
+  }
+
+  &__productCode {
+    font-size: 10px;
+    font-weight: 400;
```

Comment:
not fixed 
![image](https://github.com/mate-academy/layout_product-cards/assets/70009205/33b6cc96-c280-4aaf-bc3b-e37838eb2bc8)
```suggestion
```

Reply:
✅

Reply:
✅